_commands/tools/xclip.md

```diff
@@ -0,0 +1,56 @@
+---
+---
+
+xclip
+-------
+
+'xclip' can put the output of a command into the X-clipborad if we pipe the command into xclip.
+
+~~~ bash
+ls -la | xclip
+~~~
+
+'xclip' can also directly copy the content of the following file:
+
+~~~ bash
+xclip SomeOtherFile.txt
+~~~
+
+'xclip -o' can print the stuff in X-clipboard.
+
+~~~ bash 
+xclip -o > output.txt
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### -selection
+
+~~~ bash
+ls -la | xclip -selection clipboard
+~~~
+
+##### Break it down
+
+We can specify which X selection to use. The default is "primary". Other options are "secondary" and "clipboard". "clipboard" means the system clipboard. Thus, in the upper example, we can use Ctrl+Shift+v to paste the result later. 
```

Comment:
You can use \<kbd\>\</kbd\> to make  <kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>V</kbd>